test/unit/org/apache/cassandra/locator/AssureSufficientLiveNodesTest.java

```diff
@@ -150,7 +150,7 @@ public void addDatacenterShouldNotCausesUnavailableWithEachQuorumTest() throws T
             // alter to
             KeyspaceParams.nts(DC1, 3, DC2, 3),
             // test
-            keyspace -> ReplicaPlans.forRead(keyspace, tk, EACH_QUORUM, NeverSpeculativeRetryPolicy.INSTANCE)
+            keyspace -> ReplicaPlans.forRead(keyspace, tk, null, EACH_QUORUM, NeverSpeculativeRetryPolicy.INSTANCE)
```

Comment:
nit: Could also just have an overload of `forRead()` that keeps the old signature and passes a null index query plan through?

Reply:
I'd prefer to not do that because it adds the potential for misuse of the method. An overload would only be used for testing (not the case for `forRangeRead` but that usage is special anyway).